pkg/services/ngalert/state/manager_private_test.go

```diff
@@ -1304,6 +1304,7 @@ func TestProcessEvalResults_StateTransitions(t *testing.T) {
 									StartsAt:           t2,
 									EndsAt:             t2,
 									LastEvaluationTime: t3,
+									ResolvedAt:         t2,
```

Comment:
NOTE: this shows retaining the `ResolvedAt` time through a normal->normal transition.